### ISSUE TITLE
FEATURE REQUEST | Standard Text Imput

### ISSUE THREAD

**Issue by wayneoutthere** [Semphriss/Annotate#9]
Wow. Crazy awesome app to find today! Nice work!  Was waiting years for this...

One thing: If we could have a standard text / keyboard input method for annotations that will help with people like me who have terrible hand writing... also would allow, I think, more detailed annotations.  

For example,  you could highlight, or circle with a pencil something, then grab a 'text ABC' icon and start typing a related 'note' in small font to make better use of the space and to improve legibility

Thanks again for this!

**Comment by Semphriss** [comment]
Thanks for the feature request!

I indeed have a note on my to-do list about a typewriter tool that would do that; I'll bump it to higher priority.

**Comment by Semphriss** [comment]
I've added a very basic text editor, the release will happen in a few minutes.

Since creating the tool at all took me quite some time, I released it in a somewhat not-very-stable status, and I'll fix the remaining bugs over the next few days/weeks. It's also not very comfortable to use, I really did just the bare minimum to make it work. If you would like it to work a certain way, I've opened #11 to collect ideas for an eventual overhaul. Feel free to send any idea!
